docs/resources/lb_route.md

```diff
@@ -48,7 +50,11 @@ The following arguments are supported:
 
 - `backend_id` (Required) - The ID of the backend to which the route is associated.
 - `frontend_id`: (Required) The ID of the frontend to which the route is associated.
-- `match_sni` - (Required) The SNI to match.
+- `match` - (Required) The value to match a redirection.
```

Comment:
Please, add a message to deprecate this attribute.  Recommend the user to use the new resource or the steps to migrate to the new schema